dataverse-webapp/src/main/webapp/resources/vecler/theme-base.scss

```diff
@@ -1725,7 +1725,7 @@ div.filesTable {
 	}
 }
 
-@media screen and (min-width: $screen-md-min) {
+@media screen and (max-width: $screen-md-min) {
```

Comment:
You're change will do current things:
 - On wide device table will render in full width
 - On small device table will render in 50% of full width

I don't think that is the thing that we want to achieve.
If we think that table should always render in full width then this whole rule is unnecessary.
To manipulate whether table is scrollable we should add css class to datatable:
`no-min-width` - table will not cause scrolling to appear at all
`medium-min-width` - table will cause scrolling, but it let to be a little smaller (breakpoint for scroll has smaller `width`)
If we want some more options then introducing new class is required (analogous to medium-min-width)

Reply:
Well, I didn't know what was the idea behind this modification so I assumed that maybe the parameter was reversed. The task is also not that specific so I assumed that the whole width will be fine. So now I'm not sure whether to just remove the rule or to also take care of scrolling.

Reply:
Well, experiment with it. Table doesn't require much space so `no-min-width` maybe will be a good choice. As for if it should always display in full width - I don't have strong opinion - maybe if it causes more problems then it resolves then setting it to always display in full width is not a bad idea.

Btw. in task there is also point: `The last column should be labeled "Active" (NOT "Active by Default") - same thing with the column name on "Licenses" tab` which is not addressed in this pr

Reply:
Ok. I see that we went for full width + `no-min-width`